tests/unit/services/TemplatesService.spec.js

```diff
@@ -0,0 +1,63 @@
+import { api } from 'src/boot/axios';
+import * as TemplatesService from 'src/services/TemplatesService';
+import { getTemplatesByType } from 'src/composables/TemplateManager';
+
+jest.mock('src/boot/axios', () => ({
+  api: {
+    get: jest.fn(),
+  },
+}));
+
+jest.mock('src/composables/TemplateManager', () => ({
+  getTemplatesByType: jest.fn(),
+}));
+
+describe('Templates Service', () => {
+  describe('Test function: getApiTemplatesByType', () => {
+    it('should return a template list', async () => {
+      const resultGetTemplates = { content: [{ name: 'Project Template', type: 'PROJECT' }] };
+      api.get.mockImplementation(() => Promise.resolve(resultGetTemplates));
+
+      const res = await TemplatesService.getAPITemplatesByType('project');
+
+      expect(res.content.length).toEqual(1);
+      expect(res.content[0].name).toEqual('Project Template');
+      expect(res.content[0].type).toEqual('PROJECT');
+    });
+  });
+
+  describe('Test function: getProjectTemplates', () => {
+    const OLD_ENV = process.env;
+
+    beforeEach(() => {
```

Comment:
I think beforeEach and afterAll is useless, you can simplify by adding:
`process.env.HAS_BACKEND = true;` at the end of your first `it`

Reply:
it doesn't work.